src/components/base/outline-link/outline-link.stories.ts

```diff
@@ -1,4 +1,4 @@
-import { html, HTMLTemplateResult } from 'lit';
+import { html, TemplateResult } from 'lit-html';
```

Comment:
Is there a difference between the `html` in `lit` and `lit-html` and a reason for picking one or the other?

I noticed in one of the components that `lit-html` doesn't have `CSSResultGroup`, but `lit` does. `outline-button` uses this as an example.

Reply:
I think we are exclusively supposed to be using the `lit` package only now, however, this seems like a work around for this issue with what was going on with stories no longer rendering. @mabry1985 did you have a link to an issue in Lit (or openwc/testing) related to this?

Reply:
@himerus I'm currently in discussion with someone on the Lit slack about seeing if we can sort this out I'll open an issue if that doesn't get matters resolved. I think the workaround is okay for now though.